introduction.md

```diff
@@ -0,0 +1 @@
+Here I am, Husain Abdali from Bahrain.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
Added

Reply:
c